src/Scripts/ShowTasks.js

```diff
@@ -0,0 +1,68 @@
+function showTasks(tasks) {
```

Comment:
its better to pass the array of tasks to render here instead of passing a string and read the tasks again inside this function